docs/graphql/core/guides/postgres/import-data.rst

```diff
@@ -0,0 +1,67 @@
+.. meta::
+   :description: import data from csv in postgres
+   :keywords: hasura, docs, postgres, import, data
+
+.. _postgres_import_data_from_csv:
+
+Import data from csv into database
+===================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+-------------
+
+CSV files are plain text files that makes it easy for any developer to create and add data. 
```

Comment:
We don't need to introduce what CSV is. We should introduce the purpose of this page.